lib/plugins.js

```diff
@@ -198,6 +198,10 @@ exports.categories = {
         }
     },
     'Utility': {
+        'disinfect': {
+            url: 'https://github.com/genediazjr/disinfect',
+            description: 'Request query, payload, and params sanitation'
```

Comment:
I agree, I think it would be proper to use sanitization. I'll provide an update.